src/components/view/Contact/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+import ContactInfo from 'components/controller/ContactInfo';
+import ContactIntake from 'components/controller/ContactIntake';
+import ContactNavigation from 'components/controller/ContactNavigation';
+import ContactSearchResults from 'components/controller/ContactSearchResults';
+
+class Contact extends Component {
+
+    render() {
+
+        const { user, match } = this.props;
+
+        const urlPathUid = match && match.params && match.params.uid && match.params.uid.toLowerCase();
+        const urlPathAction = match && match.params && match.params.action && match.params.action.toLowerCase();
+    
+        return (
+            <div>
+                <ContactNavigation
+                    >
+                </ContactNavigation>
+                <div className='page'>
+                    {!urlPathAction && (
```

Comment:
I'm just wondering what's going on here. Is this conditionally rendering stuff based on whether or not the comparison is true?

Reply:
Correct